change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -855,7 +1454,57 @@ PROCEDURE READCONFIGFILESETTINGS
    WARNINGTOCLEAR=VALUE(PARAMCLEARWARNING)
    WARNINGTOSET=VALUE(PARAMSETWARNING)
 
+   LELISTINPUT=PARAMMATCHNMTYPES
+   IF LELISTINPUT="" THEN
+    LELISTINPUT="NONE"
+   CALL LISTEXPAND
```

Comment:
Consider adding `LEMAXINDEX=MAXNAMETYPES' before the `CALL LISTEXPAND` statement to keep the system from having to loop all the way to 9999 when you only need to go to MAXNAMETYPES.  

Just don't forget to reset it back to 9999 after the call:

```
LEMAXINDEX=MAXNAMETYPES
CALL LISTEXPAND
LEMAXINDEX=9999
```

Reply:
Done